### DISCUSSION TITLE
Is there any way to "overrides" the builtin "MaterialDesignFonts" resources defined in Material.Styles/Resources/Themes/FontFamily.axaml?

### DISCUSSION THREAD

**Discussion by PRNDA** [AvaloniaCommunity/Material.Avalonia#406] [Q&A]
Hey everyone,
I want to replace the default font in Material Avalonia globally. I have googled around for a while, but nothing useful has been found.

**Comment by SKProCH** [comment]
Hello. You can just override it anywhere *lower* then the MaterialTheme in Application (Avalonia should fin your resource first). E.g. inside Window, or some control inside a window. 

If you want to override it globally, you can just setup it in Application.Resources.

**Reply by PRNDA** [reply]
> Hello. You can just override it anywhere _lower_ then the MaterialTheme in Application (Avalonia should fin your resource first). E.g. inside Window, or some control inside a window.
> 
> If you want to override it globally, you can just setup it in Application.Resources.

This is not working

**Reply by SKProCH** [reply]
Probably yes, seems like this is a bug. You can track progress #407
